portfolio/src/main/webapp/style.css

```diff
@@ -45,6 +25,46 @@ body {
   max-width: 100%;
 }
 
+#navigation {
+  font-size: 5vw;
+  display:grid;
+  grid-template-columns: 1fr 1fr;
+  grid-template-rows:1fr 1fr;
+  color:white;
+}
+
+.navigation-item {
+    display:flex;
+    flex-direction:column;
+    align-items:center;
+    justify-content:center;
+    font-size:5vw;
+    cursor: pointer;
+    background: white;
+    transition: 0.5s filter ease-in-out;
+    margin:0;
+}
+
+.navigation-item:hover {
+  filter:invert(1);
+}
+
+.navigation-item[data-link="introduction"]{
```

Comment:
[use single quotation marks](https://google.github.io/styleguide/htmlcssguide.html#CSS_Quotation_Marks), here and below

Reply:
Quotes changed with commit 1840168